entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/Organization.java

```diff
@@ -52,7 +53,8 @@
   FOAF_MBOX,
   HAS_ADDRESS,
   IDENTIFIER,
-  SAME_AS
+  SAME_AS,
+  EDM_LANGUAGE
```

Comment:
we should use the officialLanguage in the data model and only use "language" in the searialization (keeping consistent the naming in the xml and json serialization)

Reply:
Renamed to LANGUAGE as agreed.